components/build-service/base/build-pipeline-config/build-pipeline-config.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: build-pipeline-config
```

Comment:
To me current name is confusing. We are listing rather `build-pipelines` than a config of a pipeline(s). Please rename CM and the file

Reply:
If we are going to have extended config (not just key-value pairs), then we should consider following [Tekton resolver structure](https://tekton.dev/docs/pipelines/resolution-getting-started/#getting-started-with-resolvers): `resolver`+`params`, where `resolver` could be:
 - `bundles`
 - `git`
 - `cluster`
 - ...

This, in turn, would require more advanced handling on Build Service side.